src/spec/methods.cr

```diff
@@ -14,8 +14,8 @@ module Spec::Methods
   # ```
   #
   # If `focus` is `true`, only this `describe`, and others marked with `focus: true`, will run.
-  def describe(description, file = __FILE__, line = __LINE__, end_line = __END_LINE__, focus : Bool = false, tags : String | Enumerable(String) | Nil = nil, &block)
-    Spec.root_context.describe(description.to_s, file, line, end_line, focus, tags, &block)
+  def describe(description, file = __FILE__, line = __LINE__, end_line = __END_LINE__, focus : Bool = false, concurrent : Bool = false, tags : String | Enumerable(String) | Nil = nil, &block)
```

Comment:
The new parameter must not be added before any existing positional parameter for backwards compatibility. Please move `concurrent` to the end. And I would prefer to make it a named parameter. There's not much sense in writing method calls with seven positional arguments.

(I also suggest to make all but `description` named parameters, but that's a story for another time)

Reply:
> And I would prefer to make it a named parameter.

What do you mean? I thought the caller determined whether something was a named parameter.

To be clear, I agree on backwards compatibility (especially since 1.0 is out), but also I wouldn't expect anyone to be writing these positionally and I certainly wouldn't want to work on code that used them that way. 😂

Regardless, IIRC with the `-j` option the `concurrent` parameter is no longer be necessary anyway.

Reply:
> What do you mean? I thought the caller determined whether something was a named parameter.

You can force an argument to only allow it being provided as a named argument.

> When a splat parameter has no name, it means no more positional arguments can be passed, and any following parameters must be passed as named arguments. For example:

```cr
# Only one positional argument allowed, y must be passed as a named argument
def foo(x, *, y)
end

foo 1        # Error, missing argument: y
foo 1, 2     # Error: wrong number of arguments (given 2, expected 1)
foo 1, y: 10 # OK
```

From: https://crystal-lang.org/reference/syntax_and_semantics/default_values_named_arguments_splats_tuples_and_overloading.html#how-call-arguments-are-matched-to-method-parameters.

Reply:
Huh, TIL. I'd always wondered why the nameless splat existed in some stdlib methods.